src/datasources/asset-utilization/helper.ts

```diff
@@ -0,0 +1,311 @@
+import {
+    AssetUtilizationHistory,
+    Interval,
+    IntervalWithHeartbeat,
+    IsPeak, SystemLinkError,
+    UtilizationTimeFrequency,
+    Weekday
+} from "./types";
+
+export const minuteInSeconds = 60 * 1000;
+export const getLocalTimezoneOffsetInHours = () => {
+    // Get the current date
+    const currentDate = new Date();
+
+    // Get the timezone offset in minutes
+    const offsetInMinutes = currentDate.getTimezoneOffset();
+
+    // Convert the offset to hours and change the sign
+    return -offsetInMinutes / 60;
+}
+
+export const convertTimeToUTC = (start: string, end: string, timeZoneOffset: number) => {
```

Comment:
I'm pretty sure there are helpers for converting to/from UTC, can we not utilize those?

Reply:
removed